lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -3803,11 +3834,12 @@ namespace Js
 
             scriptContext->GetThreadContext()->SetIsProfilingUserCode(isUserCode); // Update IsProfilingUserCode state
         }
+#endif // ENABLE_SCRIPT_PROFILING
 
         Var aReturn = NULL;
         JavascriptMethod origEntryPoint = function->GetFunctionInfo()->GetOriginalEntryPoint();
 
-        __try
+        __TRY_FINALLY_BEGIN // SEH is not guaranteed, see the implementation
```

Comment:
@agarwal-sandeep checked other tests we have and saw `returnedvaluetests1` is actually doing the same. No reason to introduce this extra test ?